More windows make bugs Sming v 4.1

Sming 4.1, Windows 10, fresh Sming instalation. 
While make v 4.0 Sming working program there is problem with axtls-8266 component

> Fetching submodule '/C/tools/sming/Sming/Components/axtls-8266/axtls-8266' ...
> Submodule path 'axtls-8266': checked out '6c3e87b307ffb67e670909a3c5ac613032eed0ba'
> Submodule path 'axtls-8266/compat': checked out 'bc2bf7b61b9295b07d2fd9b5a9bb7824b1677c85'
> error: corrupt patch at line 411
> make: *** [/C/tools/sming/Sming/Components/axtls-8266/axtls-8266/.submodule] Error 128

`make V1`

> Fetching submodule '/C/tools/sming/Sming/Components/axtls-8266/axtls-8266' ...
> cd /C/tools/sming/Sming/Components/axtls-8266 && (rm -rf axtls-8266; git submodule update --init --force --recursive axtls-8266)
> Submodule path 'axtls-8266': checked out '6c3e87b307ffb67e670909a3c5ac613032eed0ba'
> Submodule path 'axtls-8266/compat': checked out 'bc2bf7b61b9295b07d2fd9b5a9bb7824b1677c85'
> cd /C/tools/sming/Sming/Components/axtls-8266/axtls-8266 && if [ -f ../axtls-8266.patch ]; then         git apply -v ../axtls-8266.patch --ignore-whitespace --whitespace=nowarn; elif [ -f ../.patches/axtls-8266.patch ]; then        git apply -v ../.patches/axtls-8266.patch --ignore-whitespace --whitespace=nowarn; fi && if [ -d ../.patches/axtls-8266/ ]; then cp -rf ../.patches/axtls-8266/* . ; fi
> error: corrupt patch at line 411
> make: *** [/C/tools/sming/Sming/Components/axtls-8266/axtls-8266/.submodule] Error 128
>

I have really fresh Sming installation ... Windows 10 64 bit 

> C:\tools\sming\Sming>git update-git-for-windows
> Git for Windows 2.25.0.windows.1 (64bit)
> Up to date

> C:\tools\sming\Sming>git diff develop master
> fatal: ambiguous argument 'master': unknown revision or path not in the working tree.
> Use '--' to separate paths from revisions, like this:
> 'git <command> [<revision>...] -- [<file>...]'

and

> C:\tools\sming\Sming>git status
> On branch develop
> Your branch is up to date with 'origin/develop'.
> 
> Changes not staged for commit:
>   (use "git add <file>..." to update what will be committed)
>   (use "git checkout -- <file>..." to discard changes in working directory)
> 
>         modified:   ../samples/Basic_WebSkeletonApp/app/configuration.cpp
> 
> no changes added to commit (use "git add" and/or "git commit -a")

In my oppinion it is not problem with windows, git or so.

after `make dist-clean` I got like below, what mean another components are ok. @slaff I dont exactly know what do you mean "Can you paste here the rejected lines?"

> CO2_sensor: Invoking 'all' for Esp8266 (debug) architecture
> 
> Fetching submodule '/C/tools/sming/Sming/Components/rboot/rboot' ...
> Submodule path 'rboot': checked out '24f6021da82b303262a939c955f6a984bce6208f'
> Checking patch Makefile...
> Checking patch rboot.c...
> Checking patch appcode/rboot-api.c...
> Applied patch Makefile cleanly.
> Applied patch rboot.c cleanly.
> Applied patch appcode/rboot-api.c cleanly.
> 
> Fetching submodule '/C/tools/sming/Sming/Components/rboot/esptool2' ...
> Submodule path 'esptool2': checked out 'a550572b9de8c25f26b7a7675b47e58861227f64'
> Checking patch elf.h...
> Checking patch esptool2.c...
> Checking patch esptool2.h...
> Checking patch Makefile...
> Applied patch elf.h cleanly.
> Applied patch esptool2.c cleanly.
> Applied patch esptool2.h cleanly.
> Applied patch Makefile cleanly.
> 
> Fetching submodule '/C/tools/sming/Sming/Arch/Esp8266/Components/esp8266/ESP8266_NONOS_SDK' ...
> rm: cannot remove directory `ESP8266_NONOS_SDK/documents/CN': Directory not empty
> Submodule path 'ESP8266_NONOS_SDK': checked out '214129bf542c832006bd26ad0ff21a3da7465ce6'
> Checking patch lib/libc.a => lib/libc.orig.a...
> Checking patch lib/libgcc.a => lib/libgcc.orig.a...
> Applied patch lib/libc.a => lib/libc.orig.a cleanly.
> Applied patch lib/libgcc.a => lib/libgcc.orig.a cleanly.
> 
> Fetching submodule '/C/tools/sming/Sming/Arch/Esp8266/Components/driver/new-pwm' ...
> Submodule path 'new-pwm': checked out '19cb69e9ce5071686d0e2a28962fd06e11d03a61'
> Checking patch pwm.c...
> Applied patch pwm.c cleanly.
> 
> Fetching submodule '/C/tools/sming/Sming/Arch/Esp8266/Components/esptool/esptool' ...
> Submodule path 'esptool': checked out 'f2560887541414de54dbfb4368cdfc3fdd62e019'
> 
> Fetching submodule '/C/tools/sming/Sming/Arch/Esp8266/Components/esp-open-lwip/esp-open-lwip' ...
> Submodule path 'esp-open-lwip': checked out 'b9a23a6e9ef88d995ae74014e71a6a4227f61d62'
> Checking patch include/user_config.h...
> Checking patch Makefile.open...
> Checking patch include/arch/cc.h...
> Checking patch include/lwipopts.h...
> Checking patch include/lwip/ip_addr.h...
> Checking patch lwip/app/dhcpserver.c...
> Checking patch include/lwip/tcp_impl.h...
> Checking patch include/lwip/igmp.h...
> Checking patch include/lwip/mem.h...
> Checking patch include/lwip/app/espconn.h...
> Checking patch include/lwip/mdns.h...
> Checking patch lwip/app/espconn_udp.c...
> Checking patch lwip/core/dhcp.c...
> Checking patch lwip/core/ipv4/ip_addr.c...
> Checking patch lwip/core/tcp.c...
> Checking patch lwip/app/espconn.c...
> Checking patch lwip/app/espconn_tcp.c...
> Applied patch include/user_config.h cleanly.
> Applied patch Makefile.open cleanly.
> Applied patch include/arch/cc.h cleanly.
> Applied patch include/lwipopts.h cleanly.
> Applied patch include/lwip/ip_addr.h cleanly.
> Applied patch lwip/app/dhcpserver.c cleanly.
> Applied patch include/lwip/tcp_impl.h cleanly.
> Applied patch include/lwip/igmp.h cleanly.
> Applied patch include/lwip/mem.h cleanly.
> Applied patch include/lwip/app/espconn.h cleanly.
> Applied patch include/lwip/mdns.h cleanly.
> Applied patch lwip/app/espconn_udp.c cleanly.
> Applied patch lwip/core/dhcp.c cleanly.
> Applied patch lwip/core/ipv4/ip_addr.c cleanly.
> Applied patch lwip/core/tcp.c cleanly.
> Applied patch lwip/app/espconn.c cleanly.
> Applied patch lwip/app/espconn_tcp.c cleanly.
> 
> Fetching submodule '/C/tools/sming/Sming/Components/FlashString' ...
> Submodule path 'FlashString': checked out '804f22b4ea5ea7782a265ec30850a564da25ed71'
> 
> Fetching submodule '/C/tools/sming/Sming/Components/spiffs/spiffs' ...
> Submodule path 'spiffs': checked out 'ec68ba8208d7550860e4e78299d58a529b88bf85'
> Checking patch src/spiffs.h...
> Checking patch src/spiffs_hydrogen.c...
> Checking patch src/spiffs_nucleus.c...
> Applied patch src/spiffs.h cleanly.
> Applied patch src/spiffs_hydrogen.c cleanly.
> Applied patch src/spiffs_nucleus.c cleanly.
> 
> Fetching submodule '/C/tools/sming/Sming/Components/http-parser' ...
> Submodule path 'http-parser': checked out 'edeedb1b4d2f34e4c7d8045ac8b92adbc35e7ed7'
> Checking patch http_parser.c...
> Applied patch http_parser.c cleanly.
> 
> Fetching submodule '/C/tools/sming/Sming/Components/ws_parser' ...
> Submodule path 'ws_parser': checked out '5d3fa49e2e291f38340249e3150c40184244b7da'
> Checking patch ws_parser.c...
> Applied patch ws_parser.c cleanly.
> 
> Fetching submodule '/C/tools/sming/Sming/Components/mqtt-codec' ...
> Submodule path 'mqtt-codec': checked out 'b9fed830a33651dc9fd873d589ee26f85e74077b'
> Checking patch src/platform.h...
> Applied patch src/platform.h cleanly.
> 
> Fetching submodule '/C/tools/sming/Sming/Components/libyuarel' ...
> Submodule path 'libyuarel': checked out '463760fb101861d760eae9b46dbf7f8dfaa30179'
> 
> Fetching submodule '/C/tools/sming/Sming/Components/axtls-8266/axtls-8266' ...
> Submodule path 'axtls-8266': checked out '6c3e87b307ffb67e670909a3c5ac613032eed0ba'
> Submodule path 'axtls-8266/compat': checked out 'bc2bf7b61b9295b07d2fd9b5a9bb7824b1677c85'
> error: corrupt patch at line 411
> make: *** [/C/tools/sming/Sming/Components/axtls-8266/axtls-8266/.submodule] Error 128

--- a/Sming/Components/axtls-8266/axtls-8266.patch
+++ b/Sming/Components/axtls-8266/axtls-8266.patch
@@ -408,7 +408,7 @@ index d90b093..f18fbd5 100644
 +            m_putc((num <= 9) ? (num + '0') : (num + 'A' - 10));
          }
      }  
-
+ 
 diff --git a/ssl/crypto_misc.h b/ssl/crypto_misc.h
 index 02d9306..88c234f 100644
 --- a/ssl/crypto_misc.h

